signature-aggregator/config/config.go

```diff
@@ -13,6 +13,8 @@ import (
 const (
 	defaultAPIPort     = uint16(8080)
 	defaultMetricsPort = uint16(8081)
+
+	DefaultSignatureCacheSize = uint(1024 * 1024)
```

Comment:
Is there somewhere we could move this so that we don't have to duplicate it between here and the top-level config?

Reply:
Yeah, I wasn't too happy about this. But it's what's already happening here with the other config values.

And to make matters worse, there's a separate set of hardcoded "default config" values in `tests/utils/utils.go` that are used by the tests because the tests don't go through the viper procedure.

So unfortunately this value is repeated _4_ times: validator viper default, validator test default, aggregator viper default, aggregator test default.

But again, we've already got that duplication across the API port and the metrics port, so I'd prefer to defer that to after this PR. I'll raise an issue for it.